src/components/ErrorBoundary/fallbackRender.tsx

```diff
@@ -0,0 +1,14 @@
+export default function Fallback({error}: {error: Error | null}) {
+  console.log(error)
+  return (
+    <div role="alert"
+         className="flex flex-col justify-center items-center h-screen w-screen">
```

Comment:
Complex classNames could be done prettier with something like
https://css-tricks.com/css-modules-part-1-need/
Depend on approach anyway